_layouts/homepage.html

```diff
@@ -9,7 +9,7 @@
 
       <div class="row">
         <div class="col-md-12 header">
-          <img class="totally_free_software" src="assets/totally_free_software3.png" alt="Totally free software">
+          <img class="totally_free_software" src="assets/now_at_version_1_5.png" alt="Totally free software">
```

Comment:
Hank,

I would like you to undo this change, please.
My thinking about the "Now at version 1.X" message is to leave it in place for a few months and then go back to the "Totally free software" splash until the next version is published. They're different colours so the change is more likely to be noticed.

Hope that makes sense.

Chris

Reply:
Hi Chris,
I updated and uploaded the change to "totally_free_software3.png", so it should be good now.  I think what happened is that the fork that I made from Ryan's repository differed from the OpenRails repository and was pointing to the version 1.5 image.   
Henk